docs/advanced-backtesting.md

```diff
@@ -18,15 +18,15 @@ freqtrade backtesting -c <config.json> --timeframe <tf> --strategy <strategy_nam
 ```
 
 This will tell freqtrade to output a pickled dictionary of strategy, pairs and corresponding
-DataFrame of the candles that resulted in buy signals. Depending on how many buys your strategy
-makes, this file may get quite large, so periodically check your `user_data/backtest_results`
-folder to delete old exports.
+DataFrame of the candles that resulted in buy and sell signals.
```

Comment:
maybe we can upate terminology to "entry and exit" signals ? 

that feels more fitting - as in a short context, current terminology seems wrong.